collections/sliding_windows_test.ts

```diff
@@ -22,7 +22,7 @@ function slidingWindowsThrowsTest<T>(
     size: number,
     config?: { step?: number; partial?: boolean },
   ],
-  ErrorClass?: ErrorConstructor | undefined,
+  ErrorClass: ErrorConstructor,
```

Comment:
I've changed it to required, because none of the signatures of `assertThrows` function matches the way this function is called. I'm open to refactoring it to keep it optional though. It can be something like:
```ts
if (ErrorClass) {
    assertThrows(
    () => {
      slidingWindows(...input);
    },
    ErrorClass,
    msgIncludes,
    msg,
  );
} else {
  assertThrows(
    () => {
      slidingWindows(...input);
    },
    msg,
  );
}
```